.pre-commit-config.yaml

```diff
@@ -0,0 +1,73 @@
+default_stages:
```

Comment:
Please add a `Contributing` section to README.md explaining how to setup pre-commit.